SECURITY.rst

```diff
@@ -0,0 +1,125 @@
+..
+   [important] This document is reused across Astronomer OSS Integrations projects, any changes should also be applied to security docs in the other repositories
+
+Security
+________
+
+This document contains information on how to report security vulnerabilities in Astronomer Cosmos and
+how security issues reported to the Astronomer security team are handled.
+If you would like to learn more, refer to `https://www.astronomer.io/security/ <https://www.astronomer.io/security/>`_.
+
+At Astronomer, we recognize the critical nature of security and view it as a transparent and collaborative effort.
+If you have any concern about the security of any Astronomer public repository, or believe you have uncovered a vulnerability,
+please email `oss_security@astronomer.io <mailto:oss_security@astronomer.io>`_.
+
+.. warning::
+    Due to the nature of some security vulnerabilities, do not create a GitHub issue to report a vulnerability.
+
+
+Use of Email for Vulnerability Disclosure
+=========================================
+
+Only use the OSS security email to disclose security vulnerabilities.
+Astronomer does not accept bug reports, security implementation questions, or other security-related issues at this email address.
+If you are a customer of Astronomer, please reach out to your account team if you have any security-related questions or
+issues other than vulnerabilities, and they can assist you. Otherwise, this codebase is provided ‘as-is’ in accordance
+with its licensing structure.
+
+Scope
+=====
+
+When submitting vulnerabilities, please ensure that it is within the scope of the project, based on the following descriptions. Out-of-scope vulnerability reports will be ignored.
+
+In-scope
+........
+
+* Code base with tagged releases
+* When integrated as specified in the `official Astronomer Cosmos documentation <https://astronomer.github.io/astronomer-cosmos/>`_.
+
+Out-of-scope
+............
+
+* Any other codebase, including other Astronomer products
+* Astronomer.io website
+* Dependencies used in Astronomer Cosmos
+* Astronomer Cosmos when modified or run using an unintended configuration
+* Other systems integrated with or CSP systems hosting the deployment
+* Cookie transfers between browsers
+
+For other products and repositories owned by Astronomer, please refer to their specific security policy or to
+`https://www.astronomer.io/vulnerability-disclosure/ <https://www.astronomer.io/vulnerability-disclosure/>`_ for
+vulnerabilities associated with Astronomer products.
+
+Required information and how to disclose
+====================================
+
+Please send a single, plain-text (not HTML) email for each vulnerability you report.
+At a minimum, the report must include the following:
```

Comment:
```suggestion

Your written description of the vulnerability is a critical part of the initial report. You can optionally include images and videos in your initial report, but the written description of the vulnerability must includes the following information, at a minimum:
```

@tatiana - instead of adding after the list, lets put it before it. Makes the wording a little easier.